src/main/java/games/strategy/engine/message/RemoteInterfaceHelper.java

```diff
@@ -14,16 +14,13 @@
   static int getNumber(final String methodName, final Class<?>[] argTypes, final Class<?> remoteInterface) {
     final Method[] methods = remoteInterface.getMethods();
     Arrays.sort(methods, methodComparator);
-    if (logger.isLoggable(Level.FINEST)) {
-      logger.fine("Sorted methods:" + Arrays.asList(methods));
-    }
+
     for (int i = 0; i < methods.length; i++) {
       if (methods[i].getName().equals(methodName)) {
         final Class<?>[] types = methods[i].getParameterTypes();
-        // both null
```

Comment:
Is all this code really just trying to express the following?

```java
if (Arrays.equals(types, argTypes)) {
  return i;
}
```

Tests in `RemoteInterfaceHelperTest` still pass, although there's no coverage on the throw `IllegalStateException` case.

Reply:
Seems like it.

Reply:
Looks so : )

- 19f69e80643adda8dbe38d2fd6ccb9088753439f rewrites the array equality check
- 737a7cbde3f82fbe381830031bf5612eafde253e then rewrites with stream notation